deployer/scripts/util.sh

```diff
@@ -546,3 +546,23 @@ function wait_for_fluentd_to_catch_up() {
     echo END wait_for_fluentd_to_catch_up at `date -u --rfc-3339=ns`
     return $rc
 }
+
+docker_uses_journal() {
+    # note the unintuitive logic - in this case, a 0 return means true, and a 1
```

Comment:
Depending on who you are writing this for, this _is_ intuitive in that this is how Bash return codes always work ;)